src/pages/organisationUnits/list/OrganisationUnitList.tsx

```diff
@@ -233,7 +216,18 @@ export const OrganisationUnitList = () => {
                     <OrganisationUnitRow
                         key={row.id}
                         row={row}
-                        setExpanded={setExpanded}
+                        toggleShowAll={(id) =>
```

Comment:
This is new functionality: tried to implement the show-all toggle. I believe this would be mostly the same in old implementation (but instead it would update `expanded`).